### PR TITLE
NO-ISSUE: Removed --retry-all-errors due to old curl version in the CI

### PR DESCRIPTION
**Pull request by eranco74**
machine
Added retry decorator in order to overcome SSL_ERROR_SYSCALL

### CODE REVIEW

**Review by YuviGold — APPROVED** [review]

File: discovery-infra/bootstrap_in_place.py [hunk 1]
```diff
@@ -66,6 +66,7 @@ def installer_gather(ip, ssh_key, out_dir):
     os.remove(bundle_file_path) if os.path.exists(bundle_file_path) else None
 
 
+@utils.retry(exceptions=Exception, tries=3, delay=30)
```
Comment (omertuc) [inline discovery-infra/bootstrap_in_place.py:69]: # We retry both with Python and with curl because sometimes `curl` doesn't retry specific errors

**Review by osherdp — APPROVED** [review]

**Pull request merged**